aiogram/__init__.py

```diff
@@ -38,5 +38,5 @@
     'utils'
 ]
 
-__version__ = '2.3.dev1'
+__version__ = '2.4'
```

Comment:
Examples do not require version bumping, and if your example depends on specific version, you can note that in example module's docstring.

Reply:
i`m going to remove this. Dont remember why i do this